src/stdfblib/ita/CommandParser.h

```diff
@@ -0,0 +1,136 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Jose Cabral
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Jose Cabral
+ *    - initial API and implementation and/or initial documentation
+ *******************************************************************************/
+
+#ifndef _COMMAND_PARSER_H
+#define _COMMAND_PARSER_H
+
+#include "core/mgmcmd.h"
+#include "core/mgmcmdstruct.h"
+
+#include <string>
+
+class CDevice;
+
+namespace forte::command_parser {
```

Comment:
I'm not sure if this is a suitable namespace, better would be ita or something related to compliance profile for feasibility demonstrations where the format is coming from.